src/devices/CharacterLcd/LcdCharacterEncodingFactory.cs

```diff
@@ -31,12 +34,16 @@ static LcdCharacterEncodingFactory()
             // pixel row usually empty for the cursor. The added space at the top of the character helps by better supporting diacritical symbols.
             DefaultA02Map = new Dictionary<char, byte>();
 
+            // This map for instance can be found here: https://www.microchip.com/forums/m977852.aspx
+            DefaultSplC780Map = new Dictionary<char, byte>();
+
             // Inserts ASCII characters ' ' to 'z', which are common to most known character sets
             for (char c = ' '; c <= 'z'; c++)
             {
                 DefaultCustomMap.Add(c, (byte)c);
                 DefaultA00Map.Add(c, (byte)c);
                 DefaultA02Map.Add(c, (byte)c);
+                DefaultSplC780Map.Add(c, (byte)c);
```

Comment:
Why not deine the `c` as byte?

Reply:
Well, `byte c= ' '` doesn't work either, so this would just move the cast to the for loop. Not much better IMHO.